components/ai_chat/core/browser/engine/engine_consumer_conversation_api.cc

```diff
@@ -50,9 +50,8 @@ void EngineConsumerConversationAPI::GenerateRewriteSuggestion(
     GenerationDataCallback received_callback,
     GenerationCompletedCallback completed_callback) {
   std::vector<ConversationEvent> conversation = {
-      {mojom::CharacterType::HUMAN, ConversationEventType::UserText, text},
-      {mojom::CharacterType::HUMAN, ConversationEventType::RequestRewrite,
-       question}};
+      {mojom::CharacterType::HUMAN, ConversationEventType::PageExcerpt, text},
```

Comment:
Do you know if https://github.com/brave/aichat/issues/157 fix this @evq? I've just done this so I can keep testing (I'm pretty sure it isn't right).